components/sharing/SocialShareModal.vue

```diff
@@ -36,6 +68,41 @@
       mediaUrl: {
         type: String,
         default: null
+      },
+      identifier: {
+        type: String,
+        default: null
+      }
+    },
+
+    data() {
+      return {
+        oEmbedDataHtml: null,
+        embedCopied: false,
+        onItemPage: !!this.identifier
+      };
+    },
+
+    methods: {
+      requestEmbed() {
+        oEmbedForEndpoint(process.env.OEMBED_ENDPOINT || 'https://oembedjs.europeana.eu/',
+                          `${EUROPEANA_DATA_URL}/item${this.identifier}`)
+          .then((response) => {
+            if (response.data.html) {
+              this.oEmbedDataHtml = response.data.html;
+            }
+          });
+      },
+      copyEmbedCode(event) {
+        if (event.type === 'click' || event.keyCode === 13) {
```

Comment:
[keyCode is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode), so if possible we should not introduce further uses of it.

Can this instead use [key](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key)?

Reply:
Then the embed code will be copied on any key. This is for keyboard users. So if the user tabs through this module, focuses on the textarea with the embed code and hits tab again, the code will be copied. I think that would be annoying/undesired behavior.

Reply:
[Vue key modifiers](https://vuejs.org/v2/guide/events.html#Key-Modifiers) solves this and makes that conditional unnecessary :+1: